src/collections/_documentation/performance-monitoring/performance/event-detail.md

```diff
@@ -63,3 +73,90 @@ Traversing between parent and child transactions is only available on the [Busin
+
+An example of doing this in a node.js application:
+
+```javascript
+import { addGlobalEventProcessor } from '@sentry/node';
+
+addGlobalEventProcess(event => {
+  // if event is a transaction event
+  if (event.type === "transaction") {
+	event.transaction = sanitizeTransactionName(event.transaction);
+  }
+  return event;
+});
+```
+
+For browser JavaScript applications using the `Tracing` integration, the `beforeNavigate` option  can be used to better group navigation/pageload transactions together based on URL.
+
+```javascript
+import * as Sentry from '@sentry/browser';
+import { Integrations as ApmIntegrations } from '@sentry/apm';
+
+Sentry.init({
+	// ...
+  integrations: [
+    new ApmIntegrations.Tracing({
+      beforeNavigate: (location) => {        
+        // You could use your UI's routing library to find the matching
+		    // route template here. We don't have one right now, so do some basic
+		    // parameter replacements.
+        return location.pathname
+        .replace(/\d+/g, '<digits>')
+        .replace(/[a-f0-9]{32}/g, '<hash>');
+      },
+    }),
+  ],
+});
+```
+
+**Adding Query Information and Parameters to Spans**
+
+Currently, every tag has a maximum character limit of 200 characters. Tags over the 200 character limit will become truncated, losing potentially important information. To retain this data, you can split data over several tags instead.
+
+For example, a 200+ character tagged database query:
+
+`SELECT first_column, second_column, third_columns, fourth_column, fifth_column, sixth_column FROM this_is_a_long_table_name WHERE first_column=some_value AND second_column=some_other_value AND third_column=yet_another_value`
+
+The 200+ character query above will become truncated to:
+
+`SELECT first_column, second_column, third_columns, fourth_column, fifth_column, sixth_column FROM this_is_a_long_table_name WHERE first_column=some_value AND second_column=some_other_value AND third_c`
+
+Instead, using `span.set_tag` you could split the details of this query over several tags. This could be done over `columns`, `tables`, `conditions`, in this case, resulting in three different tags:
+
+```python
+import sentry_sdk
+
+...
+
+with sentry_sdk.start_span(op="db", transaction="query api") as span:
+    span.set_tag("columns", columns)
+    span.set_tag("tables", tables)
+    span.set_tag("conditions", conditions)
+    span.set_tag("query_format", query_format)
+    query_format = "SELECT {columns} FROM {tables} WHERE {conditions}"
+    query = query_format.format(
+        columns=columns,
+        tables=tables,
+        conditions=conditions,
+    )
+    ...
+```
```

Comment:
```suggestion
Instead, using `span.setTag` you could split the details of this request over several tags. This could be done over `baseUrl`, `endpoint`, `parameters`, in this case, resulting in three different tags:

```javascript
const span = transaction.startChild({
  op: "request",
  description: "setup form"
});

span.setTag("baseUrl", baseUrl);
span.setTag("endpoint", endpoint);
span.setTag("parameters", parameters);
http.get(`${base_url}/${endpoint}/`, data=parameters);
...
```
Python Version:

```suggestion
Instead, using `span.set_tag` you could split the details of this query over several tags. This could be done over `base_url`, `endpoint`, `parameters`, in this case, resulting in three different tags:

```python
import sentry_sdk

...

with sentry_sdk.start_span(op="request", transaction="setup form") as span:
    span.set_tag("base_url", base_url)
    span.set_tag("endpoint", endpoint)
    span.set_tag("parameters", parameters)
    make_request(
        "{base_url}/{endpoint}/".format(
            base_url=base_url,
            endpoint=endpoint,
        ),
        data=parameters
    )
    ...
```
```